tests/autoyast/verify_lvm_partitions.pm

```diff
@@ -0,0 +1,28 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved. This file is offered as-is,
+# without any warranty.
+#
+# Summary: Verify lvm partitions after autoyast installation
+# Maintainer: qa-sle-yast <qa-sle-yast@suse.com>
```

Comment:
s/qa-sle-yast/QA SLE YaST team/

Reply:
More importantly that e-mail address is `qa-sle-yast@suse.de` not `suse.com`.